mediorum/server/transcode.go

```diff
@@ -486,7 +520,6 @@ func (ss *MediorumServer) transcode(upload *Upload) error {
 		}
 		upload.ErrorCount = upload.ErrorCount + 1
 		ss.crud.Update(upload)
-		logger.Error("transcode error", "err", errMsg.Error())
```

Comment:
this wasn't necessary because the error gets returned on the next line and logged out later. so if the error had 100k lines then it would log 200k lines total